core/src/test/scala/org/apache/spark/h2o/ConfigurationPropertiesTestSuite.scala

```diff
@@ -158,14 +158,6 @@ class ConfigurationPropertiesTestSuite_SetNotifyLocalViaClientExtraProperties_Lo
   }
 }
 
-@RunWith(classOf[JUnitRunner])
```

Comment:
this test has no effect when running without the client ( but we still test when running in local mode which is covering very similar scenario, I think it is just fine to remove this test)